administration/console.md

```diff
@@ -16,23 +16,29 @@ The console offers the option to:
 ## Accessing the Console
 
 Accessing the console depends on the start mode of openHAB.
+
+### Started with Script
 The manually start using shell/batch script ends directly in the console.
 
-If openHAB runs a service, the console can be accessed using ssh to the openHAB host on port 8101.
+### Running as Service
+If openHAB runs a service, the console can be accessed using ssh from the openHAB host on localhost:8101. It is intentionally not available from remote hosts due to security concerns.
```

Comment:
Thanks @hakan42 ;)

You've added localhost:8101, I fear not everyone will understand that notation :-/ Do you agree "locahost, TCP port 8101" would be better?